src/stream_wrap.cc

```diff
@@ -229,6 +232,7 @@ void StreamWrap::OnReadCommon(uv_stream_t* handle,
                               const uv_buf_t* buf,
                               uv_handle_type pending) {
   StreamWrap* wrap = static_cast<StreamWrap*>(handle->data);
+  HandleScope scope(wrap->env()->isolate());
```

Comment:
Captures all handles in this libuv callback? `OnReadCommon` calls `OnRead()` which might create handles.